boot/zephyr/prj.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_MBEDTLS_CFG_FILE="config-boot.h"
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 
 CONFIG_FLASH=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
```

Comment:
It doesn't hurt anything to put the option here, so we're not tied to the timing. As long as the option name doesn't change in Zephyr, we should be good. It gives us a little more chance of not needing additional changes.